index.js

```diff
@@ -1,28 +1,31 @@
 'use strict';
 const alfy = require('alfy');
 
-alfy.fetch('https://api.npms.io/search', {
+// Do not boost exact matches by default, unless specified by the input
+const q = /boost-exact:[^\s]+/.test(alfy.input) ? alfy.input : `${alfy.input} boost-exact:false`;
```

Comment:
Indeed, I added that because @sindresorhus likes this as the default behavior.

Will update readme.